src/layouts/shared/footer.tsx

```diff
@@ -247,3 +251,13 @@ function LanguageSelections() {
     </>
   );
 }
+
+function isExternalUrl(url: string) {
+  try {
+    const currentHostname = window.location.hostname;
+    const urlHostname = new URL(url).hostname;
+    return urlHostname !== currentHostname;
+  } catch {
+    return false;
```

Comment:
Is this in a try..catch in case `window` isn't defined?

Reply:
The try..catch is there to ensure that the function does not crash in case an invalid URL is provided.